contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
   secretName: my-secret
+  configMapName: my-configmap
+  serviceName: my-service
+  osbGuid: b58d64d2-8e01-42b5-a287-bc9ee1860f9d
```

Comment:
not sure what you mean by "real-ish" in this case since I don't think we want people to set these at all. These are just GUID used for when we talk to the SB and are more internal to our stuff.  If anything it may make sense to move them into the Status section since I think I heard you say that the Status section should be read-only to normal clients while the Spec is mutable (if the resource is mutable at all).